src/core_modules/capture-ui/internal/SelectionBoxes/SingleSelectionBoxes/SingleSelectBox/singleSelectBox.module.css

```diff
@@ -22,7 +22,7 @@
 .iconContainer {
     display: flex;
     align-items: center;
-    padding: 2px 12px 2px 12px;
+    padding: 10px 10px 10px 10px;
```

Comment:
hey I know we are using 10px when we want to give paddings. There is a convention between designers and I think Joe is following it as well, where they prefer using derivatives of 4. Like 4px 8px 12px etc. The logic behind that is that it gives you more options in the long run to choose between paddings. Also is a good rule of thumb when we developers try to give padding on something that we dont have design for.

You can see if this applies for us here or if not please feel free to resolve this comment

Reply:
Thanks, this makes sense. Changing to 12.